challenge/files/09_slices.toml

```diff
@@ -0,0 +1,80 @@
+name = "Slices"
+category = "wtf"
+default_code_snippet = """
+func main() {
+	var x []int
+	x = append(x, 0)
+	x = append(x, 1)
+	x = append(x, 2)
+
+	y3 := append(x, 3)
+    y4 := append(x, 4)
+
+    fmt.Println(append(y3, y4...))
+}
+"""
+learning_advise = """
+Learn about how slices work in Go, especially how the append function can affect slice capacity and length.
+"""
+
+[[learning_links]]
+title = "Go Slices: Usage and Internals"
+url = "https://go.dev/blog/slices-intro"
+
+[[learning_links]]
+title = "Go by Example: Slices"
+url = "https://gobyexample.com/slices"
+
+[[learning_links]]
+title = "Learn Go With Tests: Arrays and Slices"
+url = "https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/arrays-and-slices"
+
+[[questions]]
+text = "Select program output"
+type = "select_answers"
+answers = [
+    { text = "[0 1 2 3 0 1 2 4]" },
+    { text = "[0 1 2 4 0 1 2 4]", code_line_ranges = [] },
+    { text = "panic: slice bounds out of range" },
+    { text = "Other" },
+]
+
+[[questions]]
+text = "What is a slice in Go?"
+type = "select_answers"
+answers = [
+    { text = "A dynamically-sized, flexible view into the elements of an array", code_line_ranges = [] },
+    { text = "A static portion of an array that cannot change in size" },
+    { text = "A key-value pair collection similar to a map" },
+    { text = "A method to iterate over collections" },
+]
+
+[[questions]]
+text = "Choose the slice capacity of variable x after all appends."
```

Comment:
Let's make it line bound answer.

"Choose the slice capacity and len of variable x after all appends."

Reply:
@y4n-k4u Sorry. No, just "Choose the slice capacity and len of variable x"